101-aci-dynamicsnav/azuredeploy.json

```diff
@@ -0,0 +1,150 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {    
+      "name": {
+          "type": "string",
+          "metadata": {
+              "description": "Name for the container group"
+          },
+          "defaultValue": "acinavcontainergroup"
+      },    
+      "navRelease": {
+          "type": "string",
+          "metadata": {
+              "description": "After the colon: Identifier for the NAV release like 2017-cu11-de or devpreview-finus (always keep 'microsoft/dynamics-nav:')"
+          },
+          "defaultValue": "microsoft/dynamics-nav:2017"
+      },    
+      "username": {        
+          "type": "string",
+          "metadata": {
+              "description": "Username for your NAV super user"
+          },
+          "defaultValue": "admin"
+      },
+      "password": {
+          "type": "string",
+          "metadata": {
+              "description": "Password for your NAV super user and your sa user on the database"
+          },
+          "defaultValue": ""
```

Comment:
remove this property, it's never valid

Reply:
But the user needs to enter something here. How do you mean "it's never valid"?

Reply:
defaultValue should be a valid value they can be used for deployment by users - an empty string will never be valid for a password (at least I hopen not)

Reply:
done